istio-manifests/frontend-gateway.yaml

```diff
@@ -12,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START servicemesh_istio_manifests_frontend_gateway_gateway_frontend_gateway]
```

Comment:
Sorry, I'm a bit confused. 😅

**Question 1:** 

This pull-request introduces region tags in very specific parts of this sample app. Is there a reason these specific places were chosen?

**Question 2:**

I'm still confused about the value this change is adding?
Could you please elaborate on what you mean by: 
> Allows for better organization on our tools!

Reply:
not all our code snippets are tracked in the devrel services. microservice-demo in particular is missing region tags all together, and we can't track our projects in the drift tools (I sent a better description with a doc that explains these efforts in more detail!)

Reply:
@NimJay the effort is just bash and YAML samples for now, as these are the most likely candidates for inclusion in the docs. that's why the whole codebase isn't region tagged - it's unlikely we'll need to embed frontend.go into the docs.